CONTRIBUTING.md

```diff
@@ -12,9 +12,10 @@ Origin (DCO). This document was created by the Linux Kernel community and is a
 simple statement that you, as a contributor, have the legal right to make the
 contribution. See the [DCO](DCO) file for details.
 
-##  Reporting a security vulnerability
+## Security Response
 
-Due to their public nature, GitHub and mailing lists are not appropriate places for reporting vulnerabilities. Please refer to CoreOS's [security disclosure][disclosure] process when reporting issues that may be security related.
+If you've found a security issue that you'd like to disclose confidentially please contact Red Hat's Product Security team.
```

Comment:
`confidentially,`

Reply:
> `confidentially,`

Fixed with ede02b4 -> dfa406b.  Want to file a patch [upstream][1]?

[1]: https://github.com/openshift/origin/blame/a0de0163af27222a31ff7fdb1106c469666d0d7d/README.md#L219-L220